lib/simple-dom/document/node.js

```diff
@@ -163,4 +162,21 @@ ChildNodes.prototype.item = function(index) {
   return child;
 };
 
+if (Object.defineProperty) {
```

Comment:
This concerns me.  We should avoid an O(n) traversal everytime someone touches `.childNodes`.  

This will probably hurt can-simple-dom performance to make worker-thread-rendering not even worth it. 

This should probably be written as an extension to can-simple-dom that only gets brought in for environments that absolutely need it.